diagnostics/ENSO_RWS/ENSO_RWS.py

```diff
@@ -0,0 +1,145 @@
+# ======================================================================
+# NOAA Model Diagnotics Task Force (MDTF) Diagnostic Driver
+#
+# This release the following Python modules: 
+#     os, glob, json, dataset, numpy, scipy, matplotlib, 
+#     networkx, warnings, numba, netcdf4, xarrays
+#
+# This package and the MDTF code package are distributed under the LGPLv3 license 
+#        (see LICENSE.txt).
+#   updated 2021-01-20
+# ======================================================================
+import os
+import os.path
+
+# ======================================================================
+# ======================================================================
+# ======================================================================`
+
+print( "Starting ENSO_RWS.py ") 
+
+
+os.environ["ENSO_RWS_WKDIR"] = os.environ["WK_DIR"]
+
+#DRB: unfortunately these don't get copied to namelist_save, which means
+#debugging requires starting from this script. To add them here requires
+#adding the POD_HOME/util path (easy, see mdtf.py) and getting the envvars
+#dict here, but currently the file is written before the pods are called.
+
+# ==================================================================================================
+
+#### 1. LEVEL_01
+print("=================================================================")
+print(" Scripts is going to calculate ENSO composites of simple variables    ")
+print("=================================================================")
+
+print("=================================================================")
+print(" General information is in README_general.docx/README_general.pdf files under")
+print("  ~/diagnostics/ENSO_RWS                                             ")
+print("=================================================================")
+
+print("=================================================================")
+print(" The LEVEL_01 routine requires the following monthly variables in the input:  ")
+print("  Zg - geopotential height [m]        ")
+print("  U  - U wind [m/s], V - V wind [m/s] ")
+print("  T  - temperature [K]                ")
+print("  OMEGA  - vertical velocity [Pa/s]   ")
+print("  TS   - skin surface temperature [K] ")
+print("  PR   - precipitation rate [kg/m2/s] ")
+print("=================================================================")
+
+
+print("=================================================================")
+print(" More detailed information regarding the LEVEL_01 module is in  ")
+print(" README_LEVEL_01.docx/README_LEVEL_01.pdf files under ~/diagnostics/ENSO_RWS/LEVEL_01/")
+print("=================================================================")
+       
+###  
+print("=================================================================")
+print(" Starting Observational LEVEL_01 module                         ")
+os.system("python "+os.environ["POD_HOME"]+"/LEVEL_01/check_input_files_OBS.py")

```

Comment:
I'm curious why these are being done as system calls.  Is there a reason they are not native Python?

Reply:
John I do not believe there is a reason for the calls.  I guess that is how I learnt it  from  some example.